flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableEnvironment.java

```diff
@@ -1040,6 +1051,17 @@ void createTemporarySystemFunction(
      */
     boolean dropTemporaryView(String path);
 
+    /**
+     * Drops a view registered in the given path.
+     *
+     * <p>Temporary objects can shadow permanent ones. If a permanent object in a given path exists,
```

Comment:
Temporary objects can shadow permanent ones. If a temporary object exists in a given path, make sure to drop the temporary object first using {@link #dropTemporaryView}. This method can only drop permanent objects.

Reply:
done